tests/DependencyInjection/Compiler/AddExtensionsPassTest.php

```diff
@@ -102,6 +100,8 @@ public function testMissingAddExtension()
             ->method('getParameterBag')
             ->willReturn($parameterBagMock);
 
+        $this->expectException(\Symfony\Component\Config\Definition\Exception\InvalidConfigurationException::class);
```

Comment:
That's probably a matter of taste, but we could now import the class as well.

Reply:
indeed, please do it.

Reply:
Done!